File: pinyougou-web-shop/src/main/webapp/js/service/seckillService.js
app.service('seckillService',function ($http) {
    
    this.search=function (page,rows,searchEntity) {
        //page,rows,$scope.searchEntity)
        return $http.post("../seckill/search.do?page="+page+"&rows="+rows,searchEntity);
    }


    this.updateStatus=function (status, ids) {
        return $http.get("../seckill/updateStatus.do?status="+status+"&ids="+ids);
    }
})

File: pinyougou-interface/src/main/java/cn/itcast/core/service/SeckillService.java
package cn.itcast.core.service;


import cn.itcast.core.pojo.seckill.SeckillOrder;
import entity.PageResult;


public interface SeckillService {
    PageResult search(Integer page, Integer rows, SeckillOrder seckillOrder);

    void updateStatus(String status, Long[] ids);
}
